### ISSUE TITLE
Problem with screen reader

### ISSUE THREAD

**Issue by Abdulaziz-Faqihi** [TelegramMessenger/Telegram-iOS#1337]
I cannot navigate between messages using a screen reader easily.

**Comment by alexoloopios** [comment]
Issue #267 is more complete and actually explains the issues, and to this day there are accessibility issues with the app.
